source/linux/Overview/GCC_ToolChain.rst

```diff
@@ -9,12 +9,22 @@ GCC ToolChain Setup
 
 Before compiling any of the sources referenced in this document, set the cross compiler paths to the toolchains packaged in the Processor SDK [Recommended] as shown below. Refer to :ref:`yocto-toolchain` section for more details on usage.
 
-.. code-block:: console
+.. ifconfig:: CONFIG_part_variant in ('AM335X', 'AM437X')
+
+   .. code-block:: console
+
+      host# export CROSS_COMPILE_64="${SDK_INSTALL_DIR}/linux-devkit/sysroots/x86_64-arago-linux/usr/bin/arm-oe-linux-gnueabi/arm-oe-linux-gnueabi-"
+      host# export SYSROOT_64="${SDK_INSTALL_DIR}/linux-devkit/sysroots/armv7at2hf-neon-oe-linux-gnueabi"
+      host# export CC_64="${CROSS_COMPILE_64}gcc --sysroot=${SYSROOT_64}"
```

Comment:
As @jeevantelukula indicated:
> For AM335x/AM437x devices, CROSS_COMPILE_64, SYSROOT_64 and ENV_SETUP_64 paths are to be modified with appropritate locations in the installer.

Reply:
There's also the issue of, none of those toolchains are 64 bit. The arm toolchain section uses the `CROSS_COMPILE_32` variable. Does the devkit do something odd and need the `*_64` variables populated regardless?

Reply:
Updated the suffixes for those toolchains.